internal/pkg_web/test2/index_golden.html_

```diff
@@ -0,0 +1 @@
+<html></html>
```

Comment:
Golden test not doing much anymore and should probably just get removed or its functionality moved to inject-html as it was testing the injector